apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -287,24 +362,24 @@ type FailedResourcePlacement struct {
 	Condition metav1.Condition `json:"condition"`
 }
 
-// ResourcePlacementConditionType defines a specific condition of a resource placement.
+// ResourcePlacementConditionType defines a specific condition of the cluster resource placement.
 // +enum
 type ResourcePlacementConditionType string
 
 const (
-	// ResourcePlacementConditionTypeScheduled indicates whether we have selected at least one resource to be placed to at least one member cluster and created work CRs under the corresponding per-cluster namespaces (i.e., fleet-member-<member-name>).
+	// ResourceScheduledConditionType indicates whether we have selected at least one resource to be placed to at least one member cluster and created work CRs under the corresponding per-cluster namespaces (i.e., fleet-member-<member-name>).
 	// Its condition status can be one of the following:
-	// - "True" means we have selected at least one resource, targeted at least one member cluster and created the work CRs.
-	// - "False" means we have selected zero resources, zero target clusters, or failed to create the work CRs.
-	// - "Unknown" otherwise.
-	ResourcePlacementConditionTypeScheduled ResourcePlacementConditionType = "Scheduled"
+	// - "True" means we have successfully scheduled the resources to fully satisfy the placement requirement.
+	// - "False" means we didn't fully satisfy the placement requirement. We will fill the Reason field.
```

Comment:
Hi Ryan! Just for clarification here, do we still keep this condition for every cluster not selected? Or is it still capped to a specific count?

Reply:
This condition is for the entire CRP

Reply:
Understood.